build/vuejs-quiz-qpp/quiz.json

```
@@ -0,0 +1,258 @@
+[
+    {
+        "Question":
+          "In CSS, Flex is a shorthand property for which of the following properties?",
```

Since these questions were pulled from the developer quiz site, prettier was set to break to a new line.
But that won't be necessary here in this repo.
Let's have everything be on the same line for consistency.

```suggestion
        "Question": "In CSS, Flex is a shorthand property for which of the following properties?",
```
 
You should make that change for all of the questions and explanation fields 👍